### PR TITLE
takes encryption controllers from apiservercontrollerset

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -106,6 +105,18 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
...
+		TargetNamespaceDeploymentInformer: kubeInformersForNamespaces.InformersFor(operatorclient.TargetNamespace).Apps().V1().Deployments(),
+		NodeInformer:                      nodeInformer,
+	}
+	deployer, err := encryptiondeployer.NewRevisionLabelPodDeployer("revision", operatorclient.TargetNamespace, kubeInformersForNamespaces, resourceSyncController, kubeClient.CoreV1(), kubeClient.CoreV1(), nodeProvider)
+	if err != nil {
+		return err
+	}
+
+	migrationClient := kubemigratorclient.NewForConfigOrDie(controllerConfig.KubeConfig)
+	migrationInformer := migrationv1alpha1informer.NewSharedInformerFactory(migrationClient, time.Minute*30)
```
Comment (sttts) [inline pkg/operator/starter.go]: where is this started?

Comment (p0lyn0mial) [reply pkg/operator/starter.go]: it's not ! fixing, thx!

**Pull request merged**